### PR TITLE
unix: setreuid/setregid

### PR DESCRIPTION
**Pull request by edigaryev**
This adds [`setreuid()`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/setreuid.html) and [`setregid()`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/setregid.html) on `unix`.

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]
Makes sense, thanks!

File: libc-test/semver/unix.txt [hunk 1]
```diff
@@ -761,6 +761,8 @@ setpgid
 setsid
 setsockopt
 setuid
+setreuid
```
Comment (devnexen) [inline libc-test/semver/unix.txt:764]: nit: ideally in alphabetical order

Comment (JohnTitor) [reply libc-test/semver/unix.txt:764]: Good catch! I'm going to fix once this is merged.

**Pull request merged**